src/index.html

```diff
@@ -7,7 +7,9 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1 class="logo">MA</h1>
+    <h1 class="title">Our phones</h1>
```

Comment:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/38065883/172476912-f0b55410-addc-425f-9cea-49ec7440af9d.png">
Still nothing happened and the error appears in the console